packages/react-components/README.md

```diff
@@ -1,6 +1,25 @@
 # @giphy/react-components
 
-React components, focused on ease-of-use and performance.
+_React components, focused on ease-of-use and performance._
+
+The GIPHY React SDK lets you and your users easily access the world’s largest GIF library anywhere on your website. Whether it’s via a comment, forum post, review, chatbot or messenger, adding GIFs to your product is a simple and easy way to boost engagement and user experience. 
+
+### GIPHY React SDK includes all the same endpoints listed in our Javascript Fetch API:
+* **Search:** Search all Giphy GIFs for a word or phrase. Punctuation will be stripped and ignored.
+* **Trending:** Fetch GIFs currently trending online. Hand curated by the Giphy editorial team. The data returned mirrors the GIFs showcased on the Giphy homepage.
+* **GIF by ID:** Fetch detailed information about a GIF by ID
+* **GIFs category and subcategory:** Fetch GIFs category and subcategory
```

Comment:
@Techgrok maybe this should be _GIFs by category and subcategory_. There is also an endpoint for categories which returns a list of category names, we wouldn't want to use that in a grid or carousel